### DISCUSSION TITLE
npm build errors out modules not found

### DISCUSSION THREAD

**Discussion by exit151** [Lissy93/dashy#754] [Help and Support]
So I posted about using the yarn on centos, seeming to work but the web config errors out, and looking for the log for dashy and it's not getting any activity, so instead of just waiting, the guide recommended trying npm as an alternative.  I installed npm and ran the commands and this is the result I get:

 __ERROR  Failed to compile with 2 errors                                                                                      3:33:50 PM

These relative modules were not found:

* ../../public/conf.yml in ./src/utils/ConfigAccumalator.js
* ../public/conf.yml in ./src/router.js
Build completed in 93.357s

 ERROR  Build failed with errors.
[importantuser@localhost dashy]$__

Hoping while the yarn crowd may not be huge, there is more widespread adoption of npm for this?  Anyone have any ideas on what's wrong and what I can do?

Thanks, 
151

Edit1:  While looking at the console, I noticed npm noted there is a minor update release out..  Knowing absolutely nothing npm I tried running npm update and I got this (seems to have tried to update dashy?)

[importantuser@localhost dashy]$ npm update
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: Dashy@2.1.0
npm ERR! Found: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^7.24.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@">= 1.6.0 < 7.0.0" from @vue/cli-plugin-eslint@4.5.18
npm ERR! node_modules/@vue/cli-plugin-eslint
npm ERR!   dev @vue/cli-plugin-eslint@"^4.5.15" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/importantuser/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/importantuser/.npm/_logs/2022-06-22T22_40_24_080Z-debug-0.log
[importantuser@localhost dashy]$

**Comment by Greeley** [comment]
Aside from the username this is the exact same issue I get.
Steps to reproduce:
- Fresh install debian
- install git, npm, and nodejs
- clone dashy
- npm install yarn

I recommend using something else, and probably just staying away from all of these JavaScript framework based solutions.
Find a real solution.
